lib/aws.js

```diff
@@ -12,7 +12,7 @@
 
 'use strict';
 
-const S3 = require('aws-sdk/clients/s3');
+const S3 = require('./../build/aws/aws-sdk').S3;
```

Comment:
The `./` is not required if it already starts with `../`.
```suggestion
const S3 = require('../build/aws/aws-sdk').S3;
```

Reply:
Ah, I always use that one again, force of habbit!